common/gsPlatformUtil.c

```diff
@@ -1559,7 +1559,7 @@ char * gsiXxteaAlg(const char *sIn, int nIn, char key[XXTEA_KEY_SIZE], int bEnc,
 	// Convert result from 32-bit words to a byte stream
 	
 	
-	oStr = (char *)gsimalloc((size_t)(4 * nIn + 1));
+	oStr = (char *)gsimalloc((4 * (size_t)nIn + 1));
```

Comment:
same as above

Reply:
duplicate of https://github.com/GameProgressive/UniSpySDK/pull/27#discussion_r674395408, will be resolved together.